css/properties/-webkit-mask-box-image.json

```diff
@@ -30,7 +30,7 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "3.2"
+              "version_added": "3.1"
             },
             "safari_ios": {
               "version_added": "3.2"
```

Comment:
This was probably wrong, just 3.2 in both Safaris. iOS 2 is more likely. Since it will look like someone tried to convey "iOS support came after macOS" with these changes, is there any way to check if iOS 3.2 is correct?

Reply:
I actually think we should probably just get rid of this entry altogether.  Taking a look at `mask-border`, it specifies `-webkit-mask-box-image` as an alternative name.  (The data in `mask-border` also says Safari 3.1 / iOS 3, which leads me to believe iOS 3.2 is incorrect.)

Reply:
This was just the first entry with iOS 3.2, there are a bunch and it would be good to look at all of them to see if the iOS data is likely to be wrong. (If left unchanged, it's more likely to look like a deliberate differences when the versions don't align.)

Reply:
I've gone through and updated all of the entries where both iOS and macOS are set to 3.2, and adjusted them!  I've spot checked a few of the entries to confirm they're supposed to be in Safari 3.1, and none of them are entries that should vary in between iOS and macOS editions.